### ISSUE TITLE
[Question] Some question about X4 strategy

### ISSUE THREAD

**Issue by Jorman** [iterativv/NostalgiaForInfinity#401]
Hi, let me premise that I am not an expert in trading and even less in the X4 strategy.
From what I understand from reading the various Issues, I understand that the strategy is based on a collection of "behaviors", I interpret this as acting on certain events, a history of events, but correct me if I am wrong.
If that's true, maybe it's convenient to share some behaviors that to an untrained eye like mine seem strange, I hope I'm doing a good thing @iterativv, or let me know if there's a better place to post the screens, also let me know if you might need them or if you don't do anything with them.
In my stupidity I happened to force exits and on a subsequent check, I realized that I could have waited for the script, although I don't know for sure.

This for example is a strange behavior for me, LTC/USDT pair on Binance
![Screenshot 2024-05-23 202617](https://github.com/iterativv/NostalgiaForInfinity/assets/5203988/670c6709-3ac4-4dac-91c7-f99faf9dcff7)
![Screenshot 2024-05-23 202647](https://github.com/iterativv/NostalgiaForInfinity/assets/5203988/c413475b-e88e-4964-a448-0641a4c9f217)
Buy before the rise -> ok
Increase position -> maybe a little late (but you can't predict the future)
But then he didn't sell anymore, he was in the positive, by a lot compared to the usual average, now instead he is at a loss and I wait for the recovery, maybe it was better to exit manually?

In this case the currency is FRONT/USDT
![Screenshot 2024-05-08 225050](https://github.com/iterativv/NostalgiaForInfinity/assets/5203988/31806cb6-5b29-4c29-9da4-6a1b2a53a6a6)
I don't remember how this pair turned out, certainly as always in profit, the only strange note is the very "late" entry on the trend

If you think it would be helpful, I can add any situations that seem strange to me.
Available to help, wouldn't know how else to do it

J

**Comment by RichRadd** [comment]
1st Trade - Entry code 120
Entry code 120 is a continuious grinding tag, It will make a certain number of trades. If you look at the current open stake on this trade you will see its ~$8USD being the minimum value allowed in freqtrade (I believe), but its basically already exited.
Using NFI you need to ignore the "Current Profit" and focus on the "Total Profit" which you will see is still in the green.
It dropped all of its stake and sold prior to the drop in the token itself.
Entry code 120 is only enabled for a set number of "trusted" coins

If you had a larger stake originally, say $100, it would have "grind purchased" another $40 and then sold $132 leaving only $8 stake.
As you're running a relatively low balance (which is smart) it only purchased $8, and couldn't sell the original buy in that sense.
Iterativ can correct me if I'm wrong.

2nd Trade, I agree that this is a pretty terrible entry in my eyes, but also impossible to detect a pump before hand, especially coins that aren't in the top 100

**Comment by Jorman** [comment]
Ciao @RichRadd 
Yes I'm testing live, with a low amount, I started with about $80, to see how it performs, the idea is to add something every month, before I had 6 slots open, then searching and reading I seem to have understood that anyway NFI uses half of the available stake, I guess to keep open the chances of recovery, so these days I decreased to 4 slots.

Yes the total profit is still green, obviously before it was at about 89 it was even greener, about 6 or 7% for the single trade, which already looked good to me, I'm used to seeing closed orders around 1.3%, so that's why I was "alarmed", now for example it drop to 2% the total profit on that trade
Not being an expert my logic is very basic, why keep funds "busy" for a long time, when maybe you can make many "small" trades in the meantime? But as I always say I have only to learn, having no experience in this regard.

Yes the representation of the second trade is not the best, however it closed in the positive, of course in hindsight everyone is capable!

That is why I ask if it is useful for the developer to have this kind of images, or other data, from the live behaviors, maybe to "calibrate" the script better

I use the recommended settings I found here on git, both the blacklist and the coin selection filters, and use 75 of them

**Comment by RichRadd** [comment]
The strategy holds on to trades and funds far longer than I think it
should, and even when it derisks, it holds on to a bit of funds and does
it's best to claw it's way back.

You'll find with this strategy that there are many many times you wish it
sold instead of holding on, including when a coin drops 40% and you lose a
lot of stake.

I would be careful and test for a long time before adding additional funds.
Potentially joining the discord to see how are people who have been using
the strategy for a long time are going with it.

**Comment by iterativv** [comment]
@Jorman Hi, yes, 120 is special trades. It's the total profit that matter there. It's going to grind (partial buys and exits), until the total profit is 7.8 or so for you.

If you exited it, it's going to pick another one. Generally after live too, 120 trades can be slow, but safe enough, with no big drawdowns. The objective is to make as much profit as the first entry. That should be 20% of the slot stake, unless that is lower than 7.8 or so, then it's the min, 7.8

120 is only for selected coins too, it's not trade to trade anything. There is a list in NFI for such coins. LTC, XLM, ALGO, ADA and so on .. good ones.

**Comment by iterativv** [comment]
Also, you can join the discord server, find the links in main page.

**Comment by Jorman** [comment]
Thank you both, like always I'll leave it running, so like you said I'll join in the discord channel.
So this's how is going the trade, for now no new entry, although you can see a couple of good spots to get in
![Screenshot 2024-05-24 125943](https://github.com/iterativv/NostalgiaForInfinity/assets/5203988/bc9631ff-ef12-41ca-8325-3945e4119f3a)

**Comment by iterativv** [comment]
Yes, for normal trades also, the first entry is 50% stake. Unless that is lower than the min allowed, then it's going to be the min.

For rebuy mode, 61-62, it's 20% of the normal stake, with chance to get rebuys if dip.

For grind mode, 120, it's special, only for selected coins, set internally to NFI. It's going to grind until enough profit. Currently 120 is set to 1 max trade per bot.

**Comment by NicoNicoNico123** [comment]
I'm noob of this strategy, can someone summary this X4strategy? Even I put into Claude that over its limit

**Comment by iterativv** [comment]
> I'm noob of this strategy, can someone summary this X4strategy? Even I put into Claude that over its limit

Well, the objective is to increase the profit. There is grinding, that is partial entries & exits. To help exit sooner a trade. Generally is strict, so not too much trading is going on normally.

**Comment by stash86** [comment]
> I'm noob of this strategy, can someone summary this X4strategy? Even I put into Claude that over its limit

It's bad and super overfitted. You can give it to a professional trader, and they won't be able to explain it to you.

**Comment by NicoNicoNico123** [comment]
> > I'm noob of this strategy, can someone summary this X4strategy? Even I put into Claude that over its limit
> 
> It's bad and super overfitted. You can give it to a professional trader, and they won't be able to explain it to you.

but there is X, X2,X3, X4, what's different?

**Comment by iterativv** [comment]
This is resolved, don't hesitate to open a new issue, thanks !
